app/views/videos/show.html.haml

```diff
@@ -11,3 +11,36 @@
         .actions
           %a.btn.btn-primary(href="") Watch Now
           %a.btn.btn-default(href="") + My Queue
+
+%section.reviews.container
+  .row
+    .col-sm-10.col-sm-offset-1
+      = form_for [@video, Review.new] do |f|
+        -token_tag
```

Comment:
Oh I didn't know that! Will change it! 
I think I've got another one somewhere else, will remove it too.